.gitlab-ci.yml

```diff
@@ -55,12 +56,14 @@ onboarding_nodejs:
         - ONBOARDING_FILTER_ENV: [dev, prod]
           ONBOARDING_FILTER_WEBLOG: [test-app-nodejs-alpine]
           SCENARIO: [CONTAINER_NOT_SUPPORTED_AUTO_INJECTION]
-
+        - ONBOARDING_FILTER_ENV: [dev, prod]
+          ONBOARDING_FILTER_WEBLOG: [test-app-nodejs-16]
+          SCENARIO: [HOST_NOT_SUPPORTED_AUTO_INJECTION]
```

Comment:
This will only start working as expected once the guardrails change is released, won't it?

Reply:
Nop, in the main pipeline the test with node16 are working (using the latest release).